R/semantic_report_constructors.R

```diff
@@ -353,10 +349,7 @@ render_semantic_report_ui <- function(validation_results,
   )
   get_semantic_report_ui(n_passes, n_fails, n_warns,
                        validation_results) %>%
-    shiny.semantic::uirender(width = "100%", height = "100%") %>%
-    htmltools::tagList(
-      htmlwidgets::onStaticRenderComplete(post_render_js)
-    )
+    shiny.semantic::uirender(width = "100%", height = "100%")
```

Comment:
Have you tried using `onRender` instead of `onStaticRenderComplete`? This is the function I'm more familiar with and it always did a job. I don't know this _static_ version, but maybe it is causing the problems.